bigbluebutton-html5/imports/ui/components/poll/component.jsx

```diff
@@ -537,11 +537,7 @@ class Poll extends Component {
             <Styled.PollOptionInput
               type="text"
               value={o.val}
-              placeholder={
-                `${i < MAX_CUSTOM_FIELDS && POLL_OPTIONS_PLACEHOLDERS[i] 
```

Comment:
after removing these lines, POLL_OPTIONS_PLACEHOLDERS is no longer needed and its declaration can be removed from lines 226-232

Reply:
Done!